new-frontiers-14/frontier-station-14
# Panic bunker print strings

## About the PR



Changes printouts when panic bunker is enabled and the server is set up to print out a reason.

Print outs now span multiple lines for readability.

## Why / Balance


Fixes #2299 
Tired of this.

## How to test


1. Host a local server, connect with one client.
2. Turn on panic bunker (`panicbunker`), make sure the reason printouts are enabled (`panicbunker_show_reason true`).
3. Connect with a new client, you should get a printout matching Image 1 in Media.
4. Disable the panic bunker (`panicbunker`).
5. Connect with your new client again.
6. Run `playtime_addoverall <yournewclient> 550`
7. Disconnect from your new client.
8. Run `golobby` in your older client, re-enable the panic bunker (`panicbunker`).
9. Connect with your new client again, you should get a printout matching Image 2 in Media.

## Media


Image 1: Failure to connect with time needed >= 60 minutes.
![failure_long](https://github.com/user-attachments/assets/6700531d-b666-4d13-b9de-9358e2652baa)

Image 2: Failure to connect with time needed < 60 minutes.
![playtime_short](https://github.com/user-attachments/assets/609397c1-1a67-437f-a1b1-02854a1f9340)

Note that somebody might get a printout that says they need 0 minutes of playtime left.  RIP.

## Requirements



## Breaking changes


**Changelog**


No in-game effect, no changelog.
